cmd/addvalidators.go

```diff
@@ -167,6 +170,12 @@ func runAddValidatorsSolo(ctx context.Context, conf addValidatorsConfig) (err er
 		return errors.Wrap(err, "transform cluster manifest")
 	}
 
+	nextKeystoreIdx := len(cluster.Validators) - conf.NumVals
```

Comment:
Add comment on this

Reply:
No, since the `nextKeystoreIdx` value is calculated after the cluster is transformed, so the cluster contains both old & new validators. Hence, needed to subtract the new validator count.

Although, since this doesn't seem to be intuitive, I'll just move this before `manifest.Transform()` is called.